### PR TITLE
trash-cli: update to 0.21.7.24.

### PR DESCRIPTION
**Pull request by TeddyDD**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by ericonr** [comment]
Some tests seem to be expecting failures due to permissions, which won't happen in our CI container. Any chance we could skip those specifically?

**Comment by TeddyDD** [comment]
I could patch decorators for failing tests

```
@pytest.mark.skip(reason="fails in CI")
```
or we could disable check altogether.

**Comment by paper42** [comment]
It is possible to deselect failing tests with `make_check_args="--deselect="tests/test_files.py"` or mark tests broken in CI.

File: srcpkgs/trash-cli/template [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
+version=0.21.7.24
 revision=1
 build_style=python3-module
-hostmakedepends="python3"
+make_check_args="--deselect='tests/test_files.py::Test_make_unreadable_file' \
+ --deselect='tests/test_files.py::Test_make_unreadable_dir' \
+ --deselect='tests/test_trash_empty.py::TestTrashEmptyCmd' \
+ --deselect='tests/test_trash_list.py::Test_describe_trash_list' \
+ --deselect='tests/test_files.py::Test_make_unreadable_dir' \
+ --deselect='tests/test_trash_empty.py::TestTrashEmptyCmd'" # broken in CI
```
Comment (ericonr) [inline srcpkgs/trash-cli/template:11]: There are still broken tests; I'd consider doing `make_check=ci-skip` and removing `make_check_args`, if the tests run successfully locally.

**Comment by tibequadorian** [comment]
New version: 0.21.10.24.

**Pull request closed without merging**